src/core/lib/gprpp/string_view.h

```diff
@@ -138,6 +123,33 @@ inline bool operator==(StringView lhs, StringView rhs) {
 
 inline bool operator!=(StringView lhs, StringView rhs) { return !(lhs == rhs); }
 
+#endif  // GRPC_USE_ABSL
+
+// Converts grpc_slice to StringView.
+inline StringView to_string_view(const grpc_slice& slice) {
```

Comment:
These function names are far too generic for top-level functions inside of the `grpc_core` namespace.  Suggest calling them something like `StringViewFromSlice()`, `StringViewToCString()`, and `StringViewCmp()`.

Reply:
Done!